specs/laravel/laravel_events_list.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: List all application's events listeners

```

Comment:
same feedback here about the workflow name including Laravel

Reply:
I think using the capital letter would be helpful here in this case

Reply:
Updated done